artemis/src/main/java/tech/pegasys/artemis/DepositCommand.java

```diff
@@ -80,45 +100,98 @@ public void generate(
               defaultValue = "1")
           int validatorCount,
       @Option(
-              names = {"--output-file", "-o"},
+              names = {"--output-path", "-o"},
+              paramLabel = "<FILE|DIR>",
               description =
-                  "File to write validator keys to. Keys are printed to std out if not specified")
-          String outputFile) {
+                  "Path to output file for unencrypted keys or output directory for encrypted keystore files. If not set, unencrypted keys will be written on standard out and encrypted keystores will be created in current directory")
+          String outputPath,
+      @Option(
+              names = {"--encrypt-keys", "-e"},
+              defaultValue = "true",
+              paramLabel = "<true|false>",
+              description = "Encrypt validator and withdrawal keys. (Default: true)",
+              arity = "1")
+          boolean encryptKeys,
+      @Option(
+              names = {"--validator-password:file"},
```

Comment:
colon in the arg name is weird. I'd have expected `--validator-password-file`. Might be possible to do it with a single `--validator-password` option - if no arg use interactive prompt, if it starts with `env:` treat as an environment variable and otherwise treat as a file path.  Not entirely sure that's a good idea though...

@jakehaugen any thoughts?

Reply:
I picked the :file :env approach from PicoCLI's sample code. We can modify them as per our needs. https://github.com/remkop/picocli/blob/master/picocli-examples/src/main/java/picocli/examples/interactive/PasswordDemo.java

Reply:
Yeah I have no idea why they thought that was normal...

Reply:
You are suggesting to use `--stdin:validator-password`, `--env:validator-password`, `--file:validator-password` as an alternative? Or simply use three mutually exclusive parameters? i.e. `--validator-password-stdin`, `--validator-password-file`, `--validator-pasword-env`?

Reply:
OR.... `--validator-password=env:VAL_PASS` approach?

Reply:
Two possible approaches:
1. Separate options.  So you'd have `--validator-password-file` and `--validator-password-env` if neither of those is specified, use the interactive prompt.

2. A single option that works out how to interpret the arg (feeling like this is going to be confusing, so leaning towards option 1).  So not specifying the option or `--validator-password` with no arg uses an interactive prompt.  `--validator-password env:SOME_VAR` would read from an environment var (the `env:` prefix is what tells us it's an env var) and `--validator-password <any-arg>` reads from a file (assuming <any-arg> doesn't start with env:).

Reply:
But you should really talk to Jake. That's why we have a UX person. :)